static/elements/admin-imports.html

```diff
@@ -1,3 +1,3 @@
 <link rel="import" href="delete-link.html">
 <link rel="import" href="chromedash-userlist.html">
-<link rel="import" href="x-meter.html">
\ No newline at end of file
+<!-- <link rel="import" href="x-meter.html"> -->
```

Comment:
Unless there's a specific benefit you see from leaving this commented out, how about just deleting it?